{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "state": "PUBLISHED",
        "cveId": "CVE-2020-25691",
        "assignerOrgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
        "assignerShortName": "redhat",
        "dateUpdated": "2024-08-04T15:40:36.586Z",
        "dateReserved": "2020-09-16T00:00:00",
        "datePublished": "2022-04-01T00:00:00"
    },
    "containers": {
        "cna": {
            "providerMetadata": {
                "orgId": "53f830b8-0a3f-465b-8143-3b8a9948e749",
                "shortName": "redhat",
                "dateUpdated": "2022-10-07T00:00:00"
            },
            "descriptions": [
                {
                    "lang": "en",
                    "value": "A flaw was found in darkhttpd. Invalid error handling allows remote attackers to cause denial-of-service by accessing a file with a large modification date. The highest threat from this vulnerability is to system availability."
                }
            ],
            "affected": [
                {
                    "vendor": "n/a",
                    "product": "darkhttpd",
                    "versions": [
                        {
                            "version": "versions up to and including darkhttpd-1.13-1",
                            "status": "affected"
                        }
                    ]
                }
            ],
            "references": [
                {
                    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1893725"
                }
            ],
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "type": "CWE",
                            "lang": "en",
                            "description": "CWE-755",
                            "cweId": "CWE-755"
                        }
                    ]
                }
            ]
        },
        "adp": [
            {
                "providerMetadata": {
                    "orgId": "af854a3a-2127-422b-91ae-364da2661108",
                    "shortName": "CVE",
                    "dateUpdated": "2024-08-04T15:40:36.586Z"
                },
                "title": "CVE Program Container",
                "references": [
                    {
                        "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1893725",
                        "tags": [
                            "x_transferred"
                        ]
                    }
                ]
            }
        ]
    }
}